gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/distributed/message/LeadNodeExecutorMsg.java

```diff
@@ -255,4 +255,9 @@ public void toData(final DataOutput out) throws IOException {
     DataSerializer.writeString(this.schema , out);
     DataSerializer.writeObject(ctx, out);
   }
+
+  public void appendFields(final StringBuilder sb) {
+    sb.append("sql: " + sql);
+    sb.append("schema: " + schema);
+  }
```

Comment:
Btw preferred way to use with StringBuilder is: sb.append("sql: ").append(sql) and not a string addition which will implicitly create another StringBuilder. Doesn't matter here for performance but will be good if we follow it as a practice.